pyvrp/tests/test_Population.py

```diff
@@ -239,7 +239,8 @@ def test_elite_individuals_are_not_purged(nb_elite: int):
         assert_(id(elite_individual) in new_individuals)
 
 
-def test_binary_tournament_ranks_by_fitness():
+@mark.parametrize("k", [2, 3])
```

Comment:
I think the test for `k=3` also succeeds if `k=2` would be used internally. Should we add another test that tests the actual frequency that individuals get selected rather than their rank? I think the expected frequency for individual $i$ with rank $r_i$ is something like $(1-\frac{r_i}{n_{pop}})^{k-1} * \frac{k \cdot n_{samples}}{n_{pop}}$.

Reply:
I've added an assert of this form, thanks!